package.json

```diff
@@ -12,12 +12,20 @@
         "babel-core": "^7.0.0-bridge.0",
         "babel-loader": "^8.0.5",
         "copy-webpack-plugin": "^4.6.0",
+        "css-loader": "^2.0.0",
         "event-hooks-webpack-plugin": "^2.1.1",
+        "extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
Since this is on webpack 4, I would probably have use https://github.com/webpack-contrib/mini-css-extract-plugin instead.

Reply:
@mbulfair The Mini CSS Extract Plugin doesn't fulfill the needs here. I actually tried for quite a while to get it working with Mini CSS Extract Plugin, but it's never as powerful as the Extract Text Webpack Plugin (as the name implies).

extract-text-webpack-plugin on version 4 is compatible with Webpack 4.

Reply:
What needs does mini fall short? Granted I've stuck with mini in all my configurations since webpack 4 came out. Just curious what it doesn't do in this case @renestalder

Reply:
CSS Mini Extract is only made for CSS extraction to files, where Extract Text Plugin is made for general text extraction to files.

But I just saw that maybe for the CSS part alone I could go with the Mini CSS Extract Plugin, just to fulfill what they recommend in the readme of the Extract Text Plugin.

> Since webpack v4 the extract-text-webpack-plugin should not be used for css. Use mini-css-extract-plugin instead.

Reply:
@mbulfair I'll check again if I get it working with CSS Mini Extract.

Reply:
@renestalder did you get a chance to update this? Also I pushed some updates to latest and will need the conflicts resolved.

Reply:
@mbulfair Not yet as this version is currently running without issues in my projects so I actually didn't have a chance/reason to try it with Mini CSS extract version.

But it's on my radar. Just couldn't justify the time to put into it yet.

Reply:
@renestalder reach out on gitter when you can.